### ISSUE TITLE
choco install python gets message: Access to the path 'C:\WINDOWS\system32\**\**.exe.ignore' is denied.

### ISSUE THREAD

**Issue by joelhandwell** [chocolatey/chocolatey-coreteampackages#249]
While running `choco install python` in Windows 10, I got following message:

```
python3 v3.5.1
WARNING: The names of some imported commands from the module 'chocolateyInstaller' include unapproved verbs that might make them less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
Downloading python3 64 bit
  from 'https://www.python.org/ftp/python/3.5.1/python-3.5.1-amd64.exe'
Progress: 100% - Saving 28.25 MB of 28.25 MB (29627072/29627072)
Download of python3Install.exe (28.25 MB) completed.
Installing python3...
python3 has been installed.
Access to the path 'C:\WINDOWS\system32\Boot\winload.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\Boot\winresume.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\difx64.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\DPTopologyApp.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\DPTopologyAppv2_0.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\GfxUIEx.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\Gfxv2_0.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\Gfxv4_0.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\igfxCUIService.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\igfxEM.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\igfxext.exe.ignore' is denied.
Access to the path 'C:\WINDOWS\system32\DriverStore\FileRepository\64dp4256.inf_amd64_9191bdaf5e95d43e\igfxHK.exe.ignore' is denied.
```

**Comment by DHowett** [comment]
`$destinationFolder` is empty [here](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/python3/tools/chocolateyInstall.ps1#L24), which causes it to spill out into `pwd`; however, I can't fathom why we need to .ignore every .exe to begin with.

If there's something somewhere that checks for the presence of a .ignore file for every executable, that seems fragile and broken.

**Comment by ferventcoder** [comment]
> If there's something somewhere that checks for the presence of a .ignore file for every executable, that seems fragile and broken.

Shim generation, a tool that puts executables on the PATH without cluttering the PATH automatically, inspects a package for executables. It also looks for shim director files such as .ignore files and .gui files to tell it to ignore. The process of doing this has been around for 5 years (pretty much all of Chocolatey's existence), which is no more fragile than finding the chocolateyInstall.ps1/chocolateyUninstall.ps1 file in the package to run.

What it seems to me is what is fragile is how this package goes about creating these ignore files (instead of just placing files in the package ahead of time like most others do). Especially since the package is having the issue (and creating ignore files is not required) and not Chocolatey itself.

**Comment by ferventcoder** [comment]
Note that Chocolatey inspects package contents ***only***, not the entire system. I'm not sure what was meant by your comment @DHowett (whole system or just local search).

**Comment by DHowett** [comment]
@ferventcoder Sorry, my comment stemmed from a misunderstanding of what's going on. I didn't intend to cast aspersions over Chocolatey.

**Comment by ferventcoder** [comment]
@DHowett I had to look up that word :D

**Comment by ferventcoder** [comment]
I fixed this is in 3.5.1.20161002 (which should be up for approval soon) - #303 has the ultimate fixes for this though.
